### DISCUSSION TITLE
<title>Web Accessibility For Everyone

### DISCUSSION THREAD

**Discussion by codelyfe** [community/community#48165] [Accessibility]
### Select Topic Area

Product Feedback

### Body

I would like to help others and the opensource community by providing a short script that will help users interact with any website. Please watch video and check out the github link under the video in description. https://m.youtube.com/watch?v=DilJPC0wN5c&feature=youtu.be

**Comment by queenofcorgis** [comment]
Hi @codelyfe , we’re glad you’re here 😀

Thanks for sharing your video with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
